scripts/manageLEcerts.js

```diff
@@ -0,0 +1,73 @@
+function manageLECerts() {
+    var resp,
+        cert_key = readFile("/tmp/privkey.url"),
+        cert     = readFile("/tmp/cert.url"),
+        chain    = readFile("/tmp/fullchain.url"),
+        secondEnvName;
+
+    secondEnvName = envName.slice(0, -1) + 2;
+
+    if (cert_key.body && chain.body && cert.body) {
+        resp = bindSSL();
+        if (resp.result != 0) return resp;
+
+        if (String(envName.slice(-1)) == 1) {
+            resp = isExtIpsExist(secondEnvName);
+            if (resp.result != 0) return resp;
+
+            if (!resp.exist) {
+                resp = jelastic.env.binder.SetExtIpCount(secondEnvName, session, "ipv4", 1, nodeGroup);
+                if (resp.result != 0) return resp;
+            }
+
+            resp = bindSSL(secondEnvName);
+            if (resp.result != 0) return resp;
+        }
+    } else {
+        return {
+            result: 99,
```

Comment:
add import com.hivext.api.Response and replace 99 with Response.ERROR_UNKNOWN
OR
use  return { type: "error", message: "Can't read SSL certificate: key=" + cert_key + " cert=" + cert + " chain=" + chain };